hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -338,7 +337,7 @@ public void markContainerUnhealthy() throws StorageContainerException {
     }
     LOG.warn("Moving container {} to state {} from state:{} Trace:{}",
             containerData.getContainerPath(), containerData.getState(),
-            prevState, StringUtils.getStackTrace(Thread.currentThread()));
+            prevState);
```

Comment:
` Trace:{}` is no longer needed.

Reply:
Ohh!! Sure yes it should be removed! Thanks for the check attila!